test/dashboard/dashboardGet.test.js

```diff
@@ -0,0 +1,115 @@
+                    connectionURI: connectionURIWithoutProtocol,
+                },
+                appInfo: {
+                    apiDomain: "api.supertokens.io",
+                    appName: "SuperTokens",
+                    websiteDomain: "supertokens.io",
+                },
+                recipeList: [
+                    Dashboard.init({
+                        apiKey: "testapikey",
+                    }),
+                    EmailPassword.init(),
+                ],
+            });
+
+            const app = express();
+
+            app.use(middleware());
+
+            app.use(errorHandler());
+
+            let response = await new Promise((res) => {
+                request(app)
+                    .get(dashboardURL)
+                    .set("Authorization", "Bearer testapikey")
+                    .end((err, response) => {
+                        if (err) {
+                            res(undefined);
+                        } else {
+                            res(response.text);
+                        }
+                    });
+            });
+
+            // checking if the original connectionURL with protocol is returned in the html response.
+            assert(response.includes(`window.connectionURI = "${connectionURI}"`));
+        });
+
+        it("Test connectionURI contains https protocol", async () => {
+            const connectionURI = "https://try.supertokens.com";
+            // removing protocol from the original connectionURI.
+            const connectionURIWithoutProtocol = connectionURI.replace("https://", "");
+
+            STExpress.init({
+                supertokens: {
+                    connectionURI: connectionURIWithoutProtocol,
+                },
+                appInfo: {
+                    apiDomain: "api.supertokens.io",
+                    appName: "SuperTokens",
+                    websiteDomain: "supertokens.io",
+                },
+                recipeList: [
+                    Dashboard.init({
+                        apiKey: "testapikey",
+                    }),
+                    EmailPassword.init(),
+                ],
+            });
+
+            const app = express();
+
+            app.use(middleware());
+
+            app.use(errorHandler());
+
+            let response = await new Promise((res) => {
+                request(app)
+                    .get(dashboardURL)
+                    .set("Authorization", "Bearer testapikey")
+                    .end((err, response) => {
+                        if (err) {
+                            res(undefined);
+                        } else {
+                            res(response.text);
+                        }
+                    });
+            });
+            // checking if the original connectionURL with protocol is returned in the html response.
+            assert(response.includes(`window.connectionURI = "${connectionURI}"`));
+        });
+    });
+});
```

Comment:
add test where you also give connectionURI that with multiple URIs and check that just the 0th index is sent to the frontend.

Reply:
done